### ISSUE TITLE
Support to custom property

### ISSUE THREAD

**Issue by rmrf1039** [n0nSmoker/SQLAlchemy-serializer#37]
Hi,

First of all, it is a very useful module. However, it will be more wonderful if it can support custom property. The `.to_dict()` seems cannot detect any property assigned from `@property`, so I have to set the key and the value in the dict manually.

Sincerely,
Edwin Lu

**Comment by n0nSmoker** [comment]
Hey, thanks this issue looks quite easy to implement.
I think you can modify this function for your purposes
```
    @property
    def serializable_keys(self):
        """
        :return: set of keys available for serialization
        """
        return {a.key for a in sql_inspect(self).mapper.attrs}
 ```
       
I cannot do it myself at the moment, mostly because I have no time and to be honest I do not use this lib for a long time.
It is open source I'll merge your pill request, just do not forget to add some tests

**Comment by n0nSmoker** [comment]
This functionality will be included in new `1.4.20` release 
Usage:
```python
# Custom serializer
from sqlalchemy_serializer import SerializerMixin

class CustomSerializerMixin(SerializerMixin):
    auto_serialize_properties = True
 
# Inline in a certain model
class Model(Base, SerializerMixin):
    auto_serialize_properties = True
```
